cli/src/commands/publish.js

```diff
@@ -229,7 +247,7 @@ const command = {
     builder: yargs =>
         yargs.options({
             apikey: {
-                alias: 'k',
+                alias: ['k', 'app-hub-api-key', 'app-hub-token', 'token'],
```

Comment:
Do all of these show up in the help text?

Reply:
```
d2-app-scripts publish

Deploy the built application to a specific DHIS2 instance

Global Options:
  -h, --help     Show help                                             [boolean]
  -v, --version  Show version number                                   [boolean]
  --verbose      Enable verbose messages                               [boolean]
  --debug        Enable debug messages                                 [boolean]
  --quiet        Enable quiet mode                                     [boolean]
  --config       Path to JSON config file

Options:
  --cwd                                     working directory to use (defaults
                                            to cwd)
  --apikey, -k, --app-hub-api-key,          The API-key to use for
  --app-hub-token, --token                  authentication              [string]
  --channel, -c                             The channel to publish the
                                            app-version to   [default: "stable"]
  --baseUrl, -b                             The base-url of the App Hub instance
                                             [default: "https://apps.dhis2.org"]
  --minDHIS2Version                         The minimum version of DHIS2 the app
                                            supports                    [string]
  --maxDHIS2Version                         The maximum version of DHIS2 the app
                                            supports                    [string]
  --appId                                   Only used with --file option. The
                                            App Hub ID for the App to publish to
                                                                        [string]
  --file                                    Path to the file to upload. This
                                            skips automatic resolution of the
                                            built app and uses this file-path to
                                            upload
  --file-version                            Only used with --file option. The
                                            semantic version of the app uploaded
                                                                        [string]
```

They do. We can [hide](https://github.com/yargs/yargs/blob/master/docs/api.md#hidekey) the keys we don't want to promote.